### ISSUE TITLE
ARM64 Support?

### ISSUE THREAD

**Issue by mobilewares** [kakone/VLC.MediaElement#81]
Hi,
Just wondering if there's any plans to get this component working with ARM64 UWP Apps? Currently using it in a couple of Apps and would be great to be able to build some ARM64 versions (as we don't want to use the ARM32 emulation for various other reasons). I believe the underlying VLC Player + SDK was updated to support ARM64 sometime last year.
thanks!

**Comment by mfkl** [comment]
>  I believe the underlying VLC Player + SDK was updated to support ARM64 sometime last year.

That's true, but AFAIK it is a win32 build not targetting UWP. It should come at some point though
